app/services/phone_formatter.rb

```diff
@@ -3,10 +3,10 @@ class PhoneFormatter
 
   def format(phone, country_code: nil)
     normalized_phone = if country_code
-                         phone&.phony_normalized(country_code: country_code)
+                         Phonelib.parse(phone, country_code)
```

Comment:
Style: You might be able to get by with
```ruby
normalized_phone = Phonelib.parse(phone, country_code || DEFAULT_COUNTRY)
```

or even (for the entire thing):
```ruby
Phonelib.parse(phone, country_code || DEFAULT_COUNTRY)&.international
```

Reply:
Good idea. I was following the previous code and didn't think to refactor.